content/docs/machine-learning/quickstart.md

```diff
@@ -0,0 +1,69 @@
+---
+sidebar_position: 2
+title: Get Started with Kubeflow
+description: Learn how to get started with Civo's Kubeflow via an example.
+---
+
+<head>
+  <title>Kubeflow quickstart | Civo Documentation</title>
+</head>
+
+In this quickstart mainly focused towards Pipelines and Notebooks, we take a look at a simple example to train and run a model using Kubeflow Pipelines and Notebooks.
+
+## Before you start
+
+Make sure you can log into your Kubeflow cluster by following the instructions at [Logging into your cluster](kubeflow-dashboard.md#logging-into-your-cluster).
+
+Make sure you can create a notebook and log into the notebook by following the instructions at [Creating a Kubeflow Notebook](creating-a-new-kubeflow-notebook.md).
+
+## Run a pipeline
+
+We will first compile a pipeline DSL that will train a model using the MNIST dataset. Notice that this pipeline mainly has the following coponents which are run: hyperparameter tuning with Katib, creating a new volume for training, run a training job and finally serve the model using KServe.
+
+Now [create a new notebook instance](creating-a-new-kubeflow-notebook.md) and run the following commands:
+
+```bash
+git clone 
```

Comment:
git clone is missing a repo address?

Reply:
Ah yes, on this I had the code pushed to the civo gitlab on the link I shared in this pr, and I wanted to ask what would the best avenue to make a public repo by Civo would be?

Reply:
Ah yes, on this I had the code pushed to the civo gitlab on the link I shared in this pr, and I wanted to ask what would the best avenue to make a public repo by Civo would be?

Reply:
Ah yes, on this I had the code pushed to the civo gitlab on the link I shared in this pr, and I wanted to ask what would the best avenue to make a public repo by Civo would be?

Reply:
Any Civo admin would be able to create a repo. I believe the plan was to have an ML examples repo.

Reply:
Got it, would you or Josh be able to do it, I do not have access to do so

Reply:
I believe Josh has the required creds. I don't.